ttnn/cpp/ttnn/operations/eltwise/binary_backward/device/binary_backward_op.hpp

```diff
@@ -41,6 +41,15 @@ enum class BinaryBackwardOpType {
     LERP_BW,
     MUL_BW,
 };
+struct BinaryBackwardFunction{
+static std::function<std::vector<ttnn::Tensor>(const Tensor&, const Tensor&, const Tensor&, const MemoryConfig&)> get_function_type1(BinaryBackwardOpType OpType);
```

Comment:
Maybe it is better to provide some typedefs for the std::functions?
It is very hard to read it.
for example 
`using FunctionType1 = std::function<std::vector<ttnn::Tensor>(const Tensor&, const Tensor&, const Tensor&, float, const MemoryConfig&)>.`
Also I see `std::vector<ttnn::tensor> or vector<optional<ttnn::tensor>> `pretty often. it would  be nice to typedef it to like using `TensorVec = [std](std::vector<ttnn::Tensor> and same for TensorOptionalVec`